### PR TITLE
HPCC-17596 Document DFUPlus listhistory and erasehistory

### PR DESCRIPTION
**Pull request by JamesDeFabia**
Signed-off-by: Jim DeFabia <jamesdefabia@lexisnexis.com>

@AttilaVamos Please review

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browers
  - [ ] The component(s) render as expected

### CODE REVIEW

File: docs/HPCCClientTools/CT_Mods/CT_Comm_Line_DFU.xml [hunk 1]
```diff
@@ -1428,6 +1429,126 @@ dfuplus action=monitor event=MyEvent ip=10.150.10.75
...
 dfuplus action=monitor event=MyEvent lfn=RTTEMP::OUT::MyFile</programlisting>
         </sect3>
+
+        <sect3 id="CLI_DFU_ListHistory">
+          <title> Listhistory Operations:</title>
+
+          <para>The listhistory operation returns the history metadata in a
+          logical file.</para>
+
+          <para>History metadata is created from a copy or remote copy
```
Comment (AttilaVamos) [inline docs/HPCCClientTools/CT_Mods/CT_Comm_Line_DFU.xml]: or spray operation

File: docs/HPCCClientTools/CT_Mods/CT_Comm_Line_DFU.xml [hunk 2]
```diff
@@ -1428,6 +1429,126 @@ dfuplus action=monitor event=MyEvent ip=10.150.10.75
...
+</programlisting>
+        </sect3>
+
+        <sect3 id="CLI_DFU_EraseHistory">
+          <title>Erasehistory Operations:</title>
+
+          <para>The erasehistory operation removes the history metadata from a
+          logical file.</para>
+
+          <para>History metadata is created from a copy or remote copy
```
Comment (AttilaVamos) [inline docs/HPCCClientTools/CT_Mods/CT_Comm_Line_DFU.xml]: or spray operation

**Review by AttilaVamos — CHANGES_REQUESTED** [review]
'Spray' operation is missing from 2 places.

**Comment by AttilaVamos** [comment]
@JamesDeFabia back to you.

**Comment by JamesDeFabia** [comment]
@AttilaVamos Edits made, back to you

File: docs/HPCCClientTools/CT_Mods/CT_Comm_Line_DFU.xml [hunk 3]
```diff
@@ -1428,6 +1429,126 @@ dfuplus action=monitor event=MyEvent ip=10.150.10.75
...
+
+                  <entry>Optional. The format of the result. Valid options
+                  are: csv, xml, json, or ascii. The default is xml.</entry>
+                </row>
+
+                <row>
+                  <entry><emphasis>csvheader</emphasis></entry>
+
+                  <entry>Optional. A boolean flag (0 | 1) indicating whether
+                  to include header information in the first row whan
```
Comment (AttilaVamos) [inline docs/HPCCClientTools/CT_Mods/CT_Comm_Line_DFU.xml]: it seems typo at the end

File: docs/HPCCClientTools/CT_Mods/CT_Comm_Line_DFU.xml [hunk 4]
```diff
@@ -1428,6 +1429,126 @@ dfuplus action=monitor event=MyEvent ip=10.150.10.75
...
+                </row>
+              </tbody>
+            </tgroup>
+          </informaltable>
+
+          <para>Example:</para>
+
+          <programlisting>dfuplus action=erasehistory lfn=progguide::exampledata::accounts_copy dstxml=c:\temp\jim.xml
+
+  // this removes the hostory metadata from the file and writes an XML file containing the following:
```
Comment (AttilaVamos) [inline docs/HPCCClientTools/CT_Mods/CT_Comm_Line_DFU.xml]: typo 'hostory'

**Review by AttilaVamos — CHANGES_REQUESTED** [review]
I found two typos, otherwise I think it is correct now.

**Comment by AttilaVamos** [comment]
@JamesDeFabia back to you.

**Comment by JamesDeFabia** [comment]
@AttilaVamos Edited, back to you

**Comment by AttilaVamos** [comment]
@JamesDeFabia Thanks, it is ready to merge.

**Pull request merged**